handsome-hedgehogs/workforce-management
# everything updated for new master, addemployee

## Status
Choose:
**READY/*

## Description
Added StartDate to CreateNew Employee and
updated Create Employee Heading

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
```
git fetch --all
git checkout wp-addemployee
```
git fetch --all
git checkout wp-addemployee
verify properly added to db

## Impacted Areas in Application
Controllers/EmployeesController
Views/Employees/Create.cshtml

## Link to Feature Ticket
#[#2](https://github.com/handsome-hedgehogs/workforce-management/issues/2)
